docker/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# Fix line endings for shell scripts
+find /workspace -type f -name "*.sh" -exec dos2unix {} + || true
```

Comment:
This is strange. If there's a problem with the line ending of any script, you should directly modify the original version in the repository; otherwise, users will see a dirty workspace when they open their development environment, which is confusing.